features/step_definitions/et3_claiments_details_page_steps.rb

```diff
@@ -0,0 +1,47 @@
+Given(/^I am on the ET3 claimants details page$$/) do
+  @respondent = FactoryBot.create(:et3_respondent)
+  @claimant = FactoryBot.create(:et3_claimant)
+  start_a_new_et3_response  
+  et3_answer_respondents_details
+  expect(claimants_details_page).to be_displayed
+end
+
+When(/^I successfully submit all the claimants details$/) do
+  et3_answer_claimants_details
+end
+
+When(/^I successfully submit required claimants details only$/) do
+  et3_answer_required_claimants_details
+end
+
+Then(/^I should be taken to the earnings and benefits page$/) do
+  expect(earnings_and_benefits_page).to be_displayed
+end
+
+When(/^I click on next without providing the required claimants details$/) do
+  earnings_and_benefits_page.next
+end
+
+Then(/^I should see the error message saying the claimants details cant be blank$/) do
+  # TODO: behaviour is incorrect
+  expect(claimants_details_page).to have_error_header
+#   expect(claimants_details_page.agree_with_employment_dates_question).to have_error_inclusion
```

Comment:
Can we please remove all unused code?

Reply:
As discussed, changes are needed, these relate to https://tools.hmcts.net/jira/browse/RST-1127